lib/wikisnakker.rb

```diff
@@ -121,7 +121,7 @@ class Item
     def self.find(ids)
       lookup = Lookup.find(ids)
       data = lookup.values
-      inflated = data.map { |rd| new(rd) }
+      inflated = data.reject { |d| d[:missing] == '' }.map { |rd| new(rd) }
```

Comment:
Ah yep, good point! Fixed in https://github.com/everypolitician/wikisnakker/pull/47/commits/ec37168ab6247a179695d55fc474b9cb8395e99b